src/csharp/Grpc.Tools/build/_protobuf/Google.Protobuf.Tools.targets

```diff
@@ -80,7 +80,7 @@
            >$(Protobuf_PackagedToolsPath)/$(Protobuf_ToolsOs)_$(Protobuf_ToolsCpu)/protoc</Protobuf_ProtocFullPath>
     </PropertyGroup>
 
-    <Error Condition=" '$(DesignTimeBuild)' != 'true' and '$(PROTOBUF_PROTOC)' == ''
+    <Error Condition=" '$(DisableGrpcDesignTimeBuild)' != 'true' and '$(PROTOBUF_PROTOC)' == ''
```

Comment:
naming: this is for protobuf code generation, so probably shouldn't have  `Grpc` in the name.
`DisableProtobufDesignTimeBuild`?

Reply:
Renamed

Reply:
FWIW, I meticulously prefixed all protoc-proper tooling lists and properties with `Protobuf_`, and gRPC-only those with `Grpc_` (or resp. `_Protobuf_` and `_Grpc_` as an indication to be treated as "private, which I understand is the convention, albeit not explicit). All of these names are dumped into the immense global namespace of a build. 

When e. g. the NuGet MSBuild tooling was introduced, all the variable there are just all random CamelCase names all across the board, short and long, and not very much in line with the previous scripts. Changes of this scale got me worried; my thought was rather let Microsoft do how they see fit, but the `Prefix_Ident` style is rarely if ever used. I do not know how you guys handle that internally, with a database of used MSBuild IDs or something, but as a 3rd-party SWE I am always mindful of possible clashes.

Should we maybe keep this convention, or we may clash soon or later. If the `_` in the middle looks too weird, then maybe it is still better to keep the `Protobuf` and `Grpc` prefixes?

(and for internal packages here we're using `Namespace::Ident` or `Namespace::_Ident` style, but I went against that for gRPC/Protobuf, for it looks unusual at best, even if complies with the MSBuild XSD).

Reply:
Good point. I can rename it to Protobuf_DisableDesignTimeBuild.